src/schema/schema_manager.py

```diff
@@ -40,6 +40,7 @@
 _memcached_client = None
 _memcached_prefix = None
 
+ONTOLOGY_API_ORGAN_TYPES_ENDPOINT = '/organs/by-code?application_context=HUBMAP'
```

Comment:
Duplicate of https://github.com/hubmapconsortium/entity-api/blob/5c3f411f58f8a9cd6bf75ef8994c31cfa4f9bf0a/src/schema/schema_constants.py#L21